client/src/components/common/Questionary/index.js

```diff
@@ -74,10 +74,9 @@ class BarBrogress extends React.Component {
         {...this.props}
       >
         <Steps current={current}>
-          {steps.map(item => {
-            console.log(item.content.props.id);
-            return <Step key={item.content.props.id} />;
-          })}
+          {steps.map(item => (
```

Comment:
Just suggestion :dango: 
```suggestion
          {steps.map({ content: { props: { id } } } => (
```

Reply:
Done